### PR TITLE
implementd structure for models schema

### PR DESCRIPTION
**Pull request by SaketRaman2002**
Implemented structure for prisma setup
Added model for student for reference.
Will assign someone to add the model schema for rest of the tables

### CODE REVIEW

**Review by GetPsyched — DISMISSED** [review]

**Review by GetPsyched — COMMENTED** [review]
Do the migrations need to be pushed? I guess yeah. But which folder are they pushed to? Can't see from phone.

**Comment by SaketRaman2002** [comment]
> Do the migrations need to be pushed? I guess yeah. But which folder are they pushed to? Can't see from phone.

its auto generated by prisma 
prisma/migrations

**Comment by SaketRaman2002** [comment]
> Do the migrations need to be pushed? I guess yeah. But which folder are they pushed to? Can't see from phone.

and yes migrations are push in my company they user to push the migrations but now they have a seprate repo for migrations.

File: .gitignore [hunk 1]
```diff
@@ -29,3 +29,8 @@ out
 # OS
 *.DS_STORE
 .DS_STORE
+.next/
```
Comment (GetPsyched) [inline .gitignore:32]: Put this along with other nextjs stuff

**Review by GetPsyched — APPROVED** [review]
Small nit, LGTM otherwise

**Pull request merged**